lib/graylog2_exceptions.rb

```diff
@@ -74,19 +79,7 @@ def clean_stack(backtrace)
     arr.join("\n")
   end
 
-  def info(err, env=nil)
-    send_to_graylog2(err, env, Logger::INFO)
-  end
-
-  def warning(err, env=nil)
-    send_to_graylog2(err, env, Logger::WARN)
-  end
-
-  def error(err, env=nil)
-    send_to_graylog2(err, env, Logger::ERROR)
-  end
-
-  def send_to_graylog2(err, env=nil, log_level=nil)
+  def send_to_graylog2(err, env = nil, log_level = nil)
     begin
       notifier = GELF::Notifier.new(@args[:hostname], @args[:port], @args[:max_chunk_size])
```

Comment:
why won't we reuse this? do we really need to initialize new instance for every message?